molo/yourwords/models.py

```diff
@@ -0,0 +1,15 @@
+from wagtail.wagtailcore.models import Page
+
+
+class Competition(Page):
```

Comment:
I think you should also add the `terms_and_conditions` page like we discussed using [this example](https://github.com/praekelt/molo/blob/develop/molo/core/models.py#L50-L57).